docs/books/nvchad/vale_nvchad.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Using vale in NvChad
+author: Steven Spencer
+contributors: Franco Colussi
+tags:
+  - vale
+  - linters
+---
+
+# `vale` in NvChad (Neovim)
+
+## Introduction
+
+`vale.sh` is one of the foremost open source projects for technical writers looking to improve their voice and style consistency. It can be used with a number of editors on nearly every major OS platform (Linux, MacOS, Windows). You can find out more about the project by heading up to [the vale.sh website](https://vale.sh/). This guide is going to walk you through adding `vale` to NvChad. Since it is included in the Mason packages for install, the process is not to difficult, although it does involve some minor editing and configuration to get things going. To be clear, NvChad is really the configuration manager for the editor Neovim, so from this point forward, we will be referencing `nvim`.
+
+## Prerequisites
+
+* Familiarity with NvChad 2.0 is helpful
+* Ability to change files from the command line by using your favorite editor. (`vi` or your favorite)
+
+## Installing `vale` using Mason
+
+Installing `vale` from within NvChad using Mason, will allow for you to stay up-to-date with the package without a whole lot of extra steps. Running Mason periodically from the within `nvim` will show you if there are updates that need to be installed and allow you to update them from there. This includes `vale` once it is installed. Let us start by running `nvim` to pull up an empty file and then get into the command mode by using <kbd>SHIFT</kbd>+<kbd>:</kbd>+Mason, which should show an interface similar to this one:
+
+![vale_mason](images/vale_mason.png)
+
+Rather than looking at the entire list of packages, let us use menu item 4 for just the linters. Press <kbd>4</kbd> and scroll down in the list until you find `vale` and with your cursor on that line, press <kbd>i</kbd> to install. Your listing should now show `vale` installed:
+
+![vale_mason_installed](images/vale_mason_installed.png)
+
+## Configuring and initializing `vale`
+
+Just having `vale` installed isn't enough. We need a couple of additional things. First, we need a `.vale.ini` file that will live in your home directory eventually, but will be in the `~/.local/share/nvim/packages/vale/` directory to start. The reason we are putting this here first is that running the command that will create the styles directory will be easier if we can be in the same directory as the binary, rather than having to call the binary from the that long path in your home directory root. Creating this file is really easy if we use the configuration utility from [the `vale.sh` website](https://vale.sh/generator). We will choose "Red Hat Documentation Style Guide" for our base style and "alex" for our supplementary style. Using "alex" is optional, but it does a good job of keeping your writing gender neutral, which is important. If you choose those options, your screen should look like this:
```

Comment:
how much smithing do you want to do? earlier the subject was "you" and in this paragraph it is mixed "we" and "you". Actually the author could examine the full document using this lens. 

*"Using 'alex' is optional, but helps you catch and fix gendered, polarizing, or race-related words, etc., which is important."